### PR TITLE
docs: slight formatting update and provide example with service mesh enabled 

### PR DESCRIPTION
**Pull request by david-yu**
Update install doc for Consul K8s CLI with example of service mesh enabled.

### CODE REVIEW

File: website/content/docs/k8s/installation/install.mdx [hunk 1]
```diff
@@ -48,10 +48,14 @@ The [Homebrew](https://brew.sh) package manager is required to complete the foll
 1. Issue the `install` subcommand to install Consul on Kubernetes:
 
    ```shell-session
-   consul-k8s install <OPTIONS>
+   $ consul-k8s install <OPTIONS>
    ```
 
-   Refer to the [Consul K8s CLI reference](/docs/k8s/k8s-cli) for details about all commands and available options.
+   Refer to the [Consul K8s CLI reference](/docs/k8s/k8s-cli) for details about all commands and available options. Without any additional options passed, the `consul-k8s` CLI will install Consul on Kubernetes with the the Helm chart default values. Below is an example that installs Consul on Kubernetes with Service Mesh and CRDs enabled.
```
Comment (karl-cardenas-coding) [inline website/content/docs/k8s/installation/install.mdx:54]:
```suggestion
   Refer to the [Consul K8s CLI reference](/docs/k8s/k8s-cli) for details about all commands and available options. Without any additional options passed, the `consul-k8s` CLI will install Consul on Kubernetes by using the Consul Helm chart's default values. Below is an example that installs Consul on Kubernetes with Service Mesh and CRDs enabled. 
```

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/572149.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 2893275e33f3d5b2e184e94d7c63f0ea496fc936 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 2893275e33f3d5b2e184e94d7c63f0ea496fc936 onto `release/1.11.x` succeeded!

**Pull request merged**